test/online/unigrid.jl

```diff
@@ -0,0 +1,10 @@
+@testset "UniGrid" begin
+    seed!(42)
+    N = 30
+    D = 3
+    nInd = 20
+    X = ColVecs(rand(D, N) * 10)
+    alg = UniGrid(nInd)
+    @test repr(alg) == "Uniform grid with side length $nInd."
+    test_Zalg(alg)
+end
```

Comment:
Could you also add some tests for `UniformGrid` ? Like for `getindex` etc...

Reply:
That was my plan for today.